### DISCUSSION TITLE
Feature Request : Keyboard shortcuts

### DISCUSSION THREAD

**Discussion by keyvanestermann** [File-New-Project/EarTrumpet#667] [Ideas]
Before EarTrumpet I was using AudioSwitcher and it had this great feature allowing us to bind hotkeys to each device, allowing us to switch from one device to another with the keyboard.

It's much quicker than opening the menu and clicking on the desired output.

**Comment by jrycp23nkt** [comment]
Yes, that would be a very good thing! With these keyboard shortcuts, this app would be close to perfection! (that and being able to install it without the Windows Store, or a portable version).

**Comment by DuckyCrayfish** [comment]
At least a shortcut to toggle through the available devices would be amazing. 

This option may be easier to implement from a settings interface perspective but may be more difficult to implement functionally.

**Comment by Ajedi32** [comment]
Dup of #211
